ivy/functional/frontends/numpy/linalg/matrix_eigenvalues.py

```diff
@@ -17,6 +17,6 @@ def eig(a):
     return ivy.eig(a)
 
 
-@from_zero_dim_arrays_to_scalar
+@to_ivy_arrays_and_back
```

Comment:
Thanks for adding the missing decorator, but I don't think you should be removing `@from_zero_dim_arrays_to_scalar`.